src/renderer.ts

```diff
@@ -25,22 +25,6 @@
  *  });
  * ```
  */
-import { createRoot } from 'react-dom/client';
-import App from './components/app';
-import './i18n';
```

Comment:
The extra work that was being done here was making it so that things like the i18n code was bundled together with the app initialization code (createRoot) and creating a cyclical dependency.

In Studio mode, Vite was importing the main chunk twice, first because is is imported on `index.html`and again when it was imported by another chunk as it added a `?v=xxxx`, making the browser treat it as a different chunk,

Doing a dynamic import forces vite to create a separate chunk.